.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+### Description:
+<!-- Add description of work done here -->
+
+### Spec:
+
+See Story: [ISSUE_NUMBER](ISSUE_URL)
```

Comment:
If you want to be really kind to future you, you can partially fill out the link text and URL. That way, you can just plug in the card number instead of copy/pasting the entire link. It could look something like this, where you replace the `iSSUE` text with an actual issue number:
`See Story: [FSA22V2-ISSUE](https://sparkbox.atlassian.net/browse/FSA22V2-ISSUE)`

This is in no way a necessary change; it's just something I find convenient on projects.

Reply:
This is great! Will do and thank you! 
:)